contranomy-sim/tests/Tests/ContranomySim/FirmwareIntegrationTests.hs

```diff
@@ -25,6 +25,7 @@ import           Paths_contranomy_sim
 import           ContranomySim.DeviceTreeCompiler
 import           ContranomySim.MemoryMapConsts
 import           System.Exit (exitFailure)
+import Numeric
```

Comment:
I know that now we don't do this much aligning with indentation anymore, but this import should either be aligned with the others or the rest should be un-aligned